app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemIdentityContent.kt

```diff
@@ -284,3 +287,32 @@ fun VaultItemIdentityContent(
         }
     }
 }
+
+@Composable
+private fun IdentityCopyField(
+    label: String,
+    value: String,
+    contentDescription: String,
```

Comment:
Should this be `copyContentDescription`?

Reply:
agree